src/runtime_src/core/edge/user/aie/common_layer/adf_runtime_api.cpp

```diff
@@ -321,6 +321,89 @@ err_code graph_api::end(unsigned long long cycleTimeout)
     return err_code::ok;
 }
 
+
+
```

Comment:
We shouldn't use camelCase in our code, but as this file comes from Compiler team, we are ok.